addon/templates/components/frost-guage.hbs

```diff
@@ -0,0 +1,36 @@
+<svg height='{{size}}' width='{{size}}' viewPort='0 0 {{size}} {{size}}' version='1.1' xmlns='http://www.w3.org/2000/svg'>
+  <circle fill='transparent' class='track' cx={{center}} cy={{center}} r={{radius}} stroke-dasharray='{{circumference}}'></circle>
+  {{#if arcs}}
+    {{#each arcs as |arc|}}
+      <circle transform='{{arc.transform}}' stroke='{{arc.color}}' fill='transparent' class='fill {{arc.classes}}' cx={{center}} cy={{center}} r={{radius}} stroke-dasharray='{{circumference}}' stroke-dashoffset='{{arc.offset}}'></circle>
```

Comment:
Definitely line wrap the properties here, probably also for the top circle (also alphabetical please)

```
<circle
  class=
  fill=
  transform=
  stroke=
```